utils/docker-tests/Dockerfile

```diff
@@ -5,5 +5,9 @@ VOLUME /payload
 RUN yum update -y && \
     yum install python-virtualenv make git -y
 
+# XXX FIXME couldn't figure out a proper way to get pip v10+ on the system
```

Comment:
Is pip v10+ really needed? If yes, I wouldn't mark it as a `FIXME`, if installation by a package manager is not a viable option, installation using `get-pip.py` script is pretty standard.

Reply:
I am just wondering why easy_install can't get the job done

Reply:
@vojtechsokol Yeah, I need at least 10.0.1 to get the job done (specifically I am interested in https://github.com/pypa/pip/commit/737dfbfc71490b41ab92c6d49b0c303c0e197f46)
@vinzenz you mean I should install pip with easy_install?